src/glfw-sandbox/engine.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+#include <windows.h>
+#include <GL/gl.h>
+#include <GL/glu.h>
+
+#define GLFW_INCLUDE_NONE
+#include <GLFW/glfw3.h> // Will give use GLFW + GL
+
+#include <iostream>
+
+
+class windowCreator
+{
+private:
+	int resWidth{};
+	int resHeight{};
+	
+	// Could you please clarify what these three do?
+	// Are these ponters? If posssible just a little clarification how this works
+	// This might be tied to main.cpp line 23-32
+	char *title;
+	GLFWmonitor *monitor;
+	GLFWwindow *share;
```

Comment:
Yes, these are pointers to the respecitve types. GLFW is C, not C++, so this is why we have this here :)

Reply:
Thank you :)